syntax = "proto3";
import "CsCmd.proto";
package pb;


enum SS_GATE_LOGIN_CMD
{
    SS_GATE_LOGIN_DEFUALT = 0;

    SS_GATE_LOGIN_LOGINAUTH_REQ = 30001;
    SS_LOGIN_GATE_LOGINAUTH_RSP = 30002;
};


message SSGate2LLoginAuthReq
{
    uint32 request_id = 1;
    LoginAuthCsReq req = 2;
}

message SSLogin2GLoginAuthRsp
{
    uint32 request_id = 1;
    LoginAuthScRsp rsp = 2;
}


